MobileWallet/Screens/Settings/SettingsParentViewController.swift

```diff
@@ -43,11 +43,34 @@ import UIKit
 class SettingsParentViewController: UIViewController {
     let navigationBar = NavigationBar()
 
+    lazy var iCloudBackup: ICloudBackup = {
```

Comment:
Is this lazy var being used anywhere?

Reply:
yes, every setting controller using it, if it needs to observe backup process